ansible_collections/arista/avd/roles/eos_l3ls_evpn/README.md

```diff
@@ -380,6 +380,9 @@ l2leaf_inband_management_vlan: < vlan_id >
 
 # QOS Profile assigned on all infrastructure links | Optional
 p2p_uplinks_qos_profile: < qos_profile_name >
+
+# Enable PTP on all infrastructure links | Optional
+p2p_uplinks_ptp_enable: < boolean | default -> false >
```

Comment:
please change to:
```yaml
p2p_uplinks_ptp:
  enable: <boolean | default -> false >
```
So we can add more knobs later

Reply:
Changed to p2p_uplinks_ptp